### DISCUSSION TITLE
Каждый день приходится подбирать новые параметры

### DISCUSSION THREAD

**Discussion by s-chernov** [hufrea/byedpi#67] [General]
Добрый день!
Ютуб с новыми параметрами 1-2 дня нормально проигрывает видео. На следующий день, каждый раз воспроизводит около 1 минуты видео и после выдаёт ошибку: "Что-то пошло не так. Обновите страницу или повторите попытку позже."

Последние использованные параметры: 
ciadpi -s 2 -d 2
ciadpi --fake 2 --split 3 --ttl 4 -M dcsmix --auto=none

ОС: Ubuntu 24.04
Интернет: мобильный оператор билайн, раздача с модема.

<details><summary>Логи</summary>
<p>

accept: fd=5
new conn: fd=6, addr=108.177.14.198:443
host: www.youtube.com (171)
split: pos=0-2 (2), m: DESYNC_SPLIT
split cancel: pos=2-2, n=2212
send: pos=2-2212
accept: fd=7
new conn: fd=8, addr=173.194.73.119:443
accept: fd=9
new conn: fd=10, addr=74.125.111.8:443
accept: fd=11
new conn: fd=12, addr=74.125.111.8:443
host: i.ytimg.com (438)
split: pos=0-2 (2), m: DESYNC_SPLIT
split cancel: pos=2-2, n=2236
send: pos=2-2236
host: rr3---sn-5goeenez.googlevideo.com (160)
split: pos=0-2 (2), m: DESYNC_SPLIT
split cancel: pos=2-2, n=1986
send: pos=2-1986
host: rr3---sn-5goeenez.googlevideo.com (192)
split: pos=0-2 (2), m: DESYNC_SPLIT
split cancel: pos=2-2, n=1986
send: pos=2-1986
accept: fd=13
accept: fd=14
new conn: fd=15, addr=128.75.235.81:443
new conn: fd=16, addr=128.75.235.81:443
host: rr6---sn-8ph2xajvh-p5gl.googlevideo.com (378)
split: pos=0-2 (2), m: DESYNC_SPLIT
split cancel: pos=2-2, n=1781
send: pos=2-1781
host: rr6---sn-8ph2xajvh-p5gl.googlevideo.com (1405)
split: pos=0-2 (2), m: DESYNC_SPLIT
split cancel: pos=2-2, n=1845
send: pos=2-1845
accept: fd=17
new conn: fd=18, addr=173.194.2.22:443
host: rr4---sn-axq7sn7s.googlevideo.com (479)
split: pos=0-2 (2), m: DESYNC_SPLIT
split cancel: pos=2-2, n=1807
send: pos=2-1807
close: fds=13,15
accept: fd=13
new conn: fd=15, addr=173.194.2.22:443
host: rr4---sn-axq7sn7s.googlevideo.com (136)
split: pos=0-2 (2), m: DESYNC_SPLIT
split cancel: pos=2-2, n=1743
send: pos=2-1743
accept: fd=19
new conn: fd=20, addr=128.75.235.81:443
host: rr6---sn-8ph2xajvh-p5gl.googlevideo.com (157)
split: pos=0-2 (2), m: DESYNC_SPLIT
split cancel: pos=2-2, n=1749
send: pos=2-1749
close: fds=14,16
accept: fd=14
new conn: fd=16, addr=128.75.235.81:443
host: rr6---sn-8ph2xajvh-p5gl.googlevideo.com (1695)
split: pos=0-2 (2), m: DESYNC_SPLIT
split cancel: pos=2-2, n=1749
send: pos=2-1749
accept: fd=21
new conn: fd=22, addr=64.233.163.100:443
host: signaler-pa.youtube.com (500)
split: pos=0-2 (2), m: DESYNC_SPLIT
split cancel: pos=2-2, n=2292
send: pos=2-2292
accept: fd=23
new conn: fd=24, addr=64.233.163.138:443
host: signaler-pa.youtube.com (1769)
split: pos=0-2 (2), m: DESYNC_SPLIT
split cancel: pos=2-2, n=2260
send: pos=2-2260
accept: fd=25
accept: fd=26
new conn: fd=27, addr=128.75.235.81:443
new conn: fd=28, addr=128.75.235.81:443
accept: fd=29
new conn: fd=30, addr=128.75.235.81:443
accept: fd=31
new conn: fd=32, addr=128.75.235.81:443
host: rr6---sn-8ph2xajvh-p5gl.googlevideo.com (1801)
split: pos=0-2 (2), m: DESYNC_SPLIT
split cancel: pos=2-2, n=1845
send: pos=2-1845
host: rr6---sn-8ph2xajvh-p5gl.googlevideo.com (179)
split: pos=0-2 (2), m: DESYNC_SPLIT
split cancel: pos=2-2, n=1781
send: pos=2-1781
host: rr6---sn-8ph2xajvh-p5gl.googlevideo.com (136)
split: pos=0-2 (2), m: DESYNC_SPLIT
split cancel: pos=2-2, n=1845
send: pos=2-1845
host: rr6---sn-8ph2xajvh-p5gl.googlevideo.com (427)
split: pos=0-2 (2), m: DESYNC_SPLIT
split cancel: pos=2-2, n=1813
send: pos=2-1813
close: fds=19,20
close: fds=14,16
close: fds=15,13
close: fds=18,17

</p>
</details>

**Comment by finn1199** [comment]
Для Linux рекомендуется добавлять параметр `--md5sig` тоже `-S`
У меня и в Arch Linux, и в MX Linux с этим параметром только нормально заработало
В результате команда запуска у меня на Ростелеком такая:
`ciadpi -i 127.0.0.1 -p 1080 -s 2 -f -1 -S`

Заметил, что если не указывать IP и порт, то чуть медленнее прогрузка происходит...
Без указания оных скрипт видимо автоматом ищет их и чуть дольше на это уходит времени.

**Reply by s-chernov** [reply]
> Для Linux рекомендуется добавлять параметр `--md5sig` тоже `-S` У меня и в Arch Linux, и в MX Linux с этим параметром только нормально заработало В результате команда запуска у меня на Ростелеком такая: `ciadpi -i 127.0.0.1 -p 1080 -s 2 -f -1 -S`
> 
> Заметил, что если не указывать IP и порт, то чуть медленнее прогрузка происходит... Без указания оных скрипт видимо автоматом ищет их и чуть дольше на это уходит времени.

К сожалению, не помогло.

**Reply by finn1199** [reply]
> К сожалению, не помогло.

А так?
`ciadpi -i 127.0.0.1 -p 1080 -s 2 -M h,d -A torst -T 3 -f -1 -r 3+s -S`

**Reply by ropucyka** [reply]
Да блин, что вот значит fake -1 а не --fake 2 например?
И можно ли где-то увидеть как прописан или предоставлен кастомный --fake вместо стандартного?

**Comment by s-chernov** [comment]
Помогло установка 1.1.1.1 в качестве безопасного dns в браузере.
